app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx

```diff
@@ -86,13 +86,13 @@ const inviteFormSelector = formValueSelector('inviteUserForm');
   (state, ownProps) => ({
     selectedProject: ownProps.data.isProjectSelector
       ? inviteFormSelector(state, 'project')
-      : projectIdSelector(state),
+      : urlProjectKeySelector(state),
     selectedUser: inviteFormSelector(state, 'user'),
     isAdmin: isAdminSelector(state),
     projectKey: projectKeySelector(state),
     initialValues: {
       role: DEFAULT_PROJECT_ROLE,
-      project: projectIdSelector(state),
+      project: urlProjectKeySelector(state),
```

Comment:
Please leave a comment here to explain why we use two different project key selectors in this case.

Reply:
You are right. It is unnecessary to use two selectors. I will remove projectKeySelector